packages/terra-form-select/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 6.54.0 - (January 4, 2024)
```

Comment:
```suggestion
## 6.53.1 - (January 4, 2024)
```

Reply:
Since it is bug fix it should be released as patch version not minor version

Reply:
Updated - https://github.com/cerner/terra-core/pull/4005/commits/1b3b3da8c0d050204dc707d14a89c608205bc4a2